html/modules/custom/reliefweb_import/src/Drush/Commands/ReliefwebImport.php

```diff
@@ -86,9 +86,22 @@ public function importWorkdayJobs(int $limit = 50): void {
         '@name' => $config['name'] ?? $name,
       ]));
 
-      $this->workdayJobImporter->setSettings($config);
-      $this->workdayJobImporter->importJobs($limit);
+      try {
+        $this->workdayJobImporter->setSettings($config);
+        $this->workdayJobImporter->validateSettings();
+        $this->workdayJobImporter->importJobs($limit);
+      }
+      catch (\Exception $e) {
+        $this->logger()->error('Import complete: something went wrong.');
```

Comment:
Why 2 error log messages? Also this error message is strange. It says import complete but the loop continues to the next tenant.

Reply:
Was added so Jenkins can check for it, I'll remove it for now since it's confusing